bundles/org.openhab.binding.icalendar/README.md

```diff
@@ -23,20 +25,54 @@ Each `calendar` thing requires the following configuration parameters:
+| `textEventValue` | The text to filter events with.                                                                                                                                                              | optional                                   |
+| `textValueType`  | The type of the text to filter with. Valid values: `TEXT` (field must contain value), `REGEX` (field must match value, completely, dot matches all, case insensetive).                       | optional/required for text-based filtering |
+
 ## Channels
 
-The channels describe the current and the next forthcoming event.
+The channels of `calendar` describe the current and the next forthcoming event.
 They are all read-only.
 
-| Channel           | Type      | Description                                                                    |
-|-------------------|-----------|--------------------------------------------------------------------------------|
-| current_presence  | Switch    | Current presence of an event, `ON` if there is currently an event, `OFF` otherwise |
-| current_title     | String    | Title of a currently present event                                             |
-| current_start     | DateTime  | Start of a currently present event                                             |
-| current_end       | DateTime  | End of a currently present event                                               |
-| next_title        | String    | Title of the next event                                                        |
-| next_start        | DateTime  | Start of the next event                                                        |
-| next_end          | DateTime  | End of the next event                                                          |
+| Channel           | Type      | Description                                                                         |
+|-------------------|-----------|-------------------------------------------------------------------------------------|
+| current_presence  | Switch    | Current presence of an event, `ON` if there is currently an event, `OFF` otherwise  |
+| current_title     | String    | Title of a currently present event                                                  |
+| current_start     | DateTime  | Start of a currently present event                                                  |
+| current_end       | DateTime  | End of a currently present event                                                    |
+| next_title        | String    | Title of the next event                                                             |
+| next_start        | DateTime  | Start of the next event                                                             |
+| next_end          | DateTime  | End of the next event                                                               |
+
+The channels of `eventfilter` are generated using following scheme, all are read-only.
```

Comment:
Ditto sub title ###Channels for "eventfilter"

Reply:
OK